includes/views/dashboard/index.php

```diff
@@ -24,7 +24,7 @@
 		<?php echo esc_html( get_admin_page_title() ); ?>
 	</h2>
 
-	<div class="flex flex-wrap -mx-2 mt-4">
+	<div class="tw-flex tw-flex-wrap -tw-mx-2 tw-mt-4">
```

Comment:
`-tw-mx-2` is not working here it breaks the layout!

these is a visible bug easily noticeable